data/indonesia/bbc_indonesian_radio/liveradio.json

```diff
@@ -1,7 +1,12 @@
 {
     "metadata": {
         "id": "urn:bbc:ares::asset:indonesia/bbc_indonesian_radio/liveradio",
-        "tags": {}
+        "tags": {},
+        "language": "id-ID",
+        "analyticsLabels": {
+            "pageIdentifier": "indonesia.bbc_indonesian_radio.liveradio.page",
+            "pageTitle": "BBC Indonesia Radio - BBC News Indonesia"
```

Comment:
Have we made this field up? 

It seems that is is being used to populate the `atiPageViewParams` which for both the index and article  use the `promo.headlines.seoHeadline` from the Ares payload.

Reply:
Yep - was some back and forward with Iain on whether or not we should have the `promo.headline` object - apparantly it's specific to CPS?  It also doesn't make sense semantically for live radio pages - there is no "headline"

If necessary, we can fix this once the ares payload is confirmed though